acceptance/scripts/generic/testrun-full.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -x
+
+# static fixtures
+REDIS_HOSTNAME=redis.delivery.puppetlabs.net
```

Comment:
Ah right, it was a copy-pasta error since I based much of this script on one from another project where I had to install the latest version of PE listed in redis.